starter_code/src/movies.js

```diff
@@ -1,6 +1,19 @@
 /* eslint no-restricted-globals: 'off' */
 // Turn duration of the movies from hours to minutes 
 
+function turnHoursToMinutes(duration) {
+    var splitDuration = duration.split("h ");
+    var hours = 60 * parseInt(splitDuration[0]);
+    var splitMinutes = parseInt(splitDuration[1].slice(0,2));
+    var totalInMinutes = hours + splitMinutes;
+   return String(totalInMinutes);
```

Comment:
My mistake, it should be a Number, not a String, so you can remove the `String` function